pkg/receiver/smartagent/README.md

```diff
@@ -2,50 +2,49 @@
-functional [Smart Agent release bundle](https://github.com/signalfx/signalfx-agent/releases/latest) on your system,
-which is already provided for x86_64/amd64 Splunk Distribution of OpenTelemetry Collector
-[installation paths](https://github.com/signalfx/splunk-otel-collector#getting-started).
+The Smart Agent Receiver allows you to use [SignalFx Smart Agent monitors](https://github.com/signalfx/signalfx-agent#monitors)
+in the [Splunk Distribution of OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector). It requires a
+[Smart Agent release bundle](https://github.com/signalfx/signalfx-agent/releases/latest),
+which is installed by the Splunk Distribution of OpenTelemetry Collector.
 
-**Beta: All Smart Agent monitors are supported at this time.  Configuration and behavior may change without notice.**
+While this document outlines some of the capabilities of the Smart Agent receiver, please see the
+[migration guide](../../../docs/signalfx-smart-agent-migration.md)
+for more information about migrating from the Smart Agent to the Splunk Distribution of the OpenTelemetry Collector.
+
+**Beta: All Smart Agent monitors are supported at this time. Configuration and behavior may change without notice.**
 
 ## Configuration
 
-Each `smartagent` receiver configuration acts as a drop-in replacement for each supported Smart Agent Monitor
-[configuration](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitor-config.md) with some exceptions:
+For each Smart Agent 
+[monitor](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitor-config.md)
+you want to put into the Collector, add a `smartagent` receiver configuration block. Once configured in the Collector, each
+`smartagent` receiver will act as a drop-in replacement for its corresponding Smart Agent monitor.
 
-1. Any Agent global or collectd desired configuration should be performed via the
-[Smart Agent Extension](../../extension/smartagentextension/README.md).
-1. In lieu of `discoveryRule` support, the Collector's
-[`receivercreator`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md)
-and associated [Observer extensions](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/README.md)
-should be used.
-1. The [`signalfx-forwarder`](https://github.com/signalfx/signalfx-agent/blob/main/docs/monitors/signalfx-forwarder.md)
-monitor should be made part of both `metrics` and `traces` pipelines utilizing the
-[`signalfx`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
-and [`sapm`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sapmexporter/README.md) exporters, respectively.
-1. All metric content replacement and transformation rules should utilize existing
+1. Put any Smart Agent or collectd configuration into the
```

Comment:
I think stressing "global" is helpful here since "any smart agent or collectd configuration" isn't accurate if users assume an implicit monitor inclusion.

Reply:
Sure, done.